rpc/api/utils.go

```diff
@@ -99,6 +98,7 @@ var (
 			"sendRawTransaction",
 			"sendTransaction",
 			"sign",
+			"syncing",
```

Comment:
It **is** there https://github.com/karalabe/go-ethereum/blob/standard-sync-stats/rpc/api/utils.go#L101